Common/Utils.ts

```diff
@@ -1,3 +1,26 @@
-export function getId():string{
-    return new Date().toISOString() + Math.random().toString().substr(2,8);
+import {PhoneNumberUtil} from 'google-libphonenumber';
+import {Guid} from 'guid-typescript';
+import {isNumberInDatabase} from '../DataAccess/contactList';
+
+export function getUserId() {
+  let id = Guid.create();
+  return id;
+}
+export async function isPhoneValid(
+  phonenumber: string,
+  userTemp: string
+): Promise<boolean> {
+  const phoneUtil = PhoneNumberUtil.getInstance();
+  const number = phoneUtil.parseAndKeepRawInput(phonenumber, 'ZA');
```

Comment:
much better👍, you'll probably want to catch exceptions when using this library to parse user input.